runtime/sema/nominal_type.go

```diff
@@ -109,3 +111,11 @@ func (t *NominalType) initializeMembers() {
 		t.members = withBuiltinMembers(t, members)
 	})
 }
+
+func (t *NominalType) isContainerType() bool {
+	return t.NestedTypes != nil
```

Comment:
Should this be as:
```suggestion
	return t.IsContainerType
```

Reply:
Or perhaps, we don't need `IsContainerType` at all, can simply rely on the `t.NestedTypes`. But then we might have to check both nil and emptiness.

Reply:
Oh, right, I think I had added `IsContainerType` at first, then changed my mind mid-way and thought we could just rely on `NestedTypes != nil` and didn't remove `IsContainerType`. 

Good catch!

Reply:
Removed the unused field in 67f13e7d1723c914c3e15a13fec63ad4e1e5d44a